src/main/java/com/bravo/user/service/UserService.java

```diff
@@ -49,7 +58,22 @@ public UserReadDto retrieve(final String id){
     LOGGER.info("found user '{}'", id);
     return resourceMapper.convertUser(user);
   }
-
+  
+  public PasswordValidateDto validate(final PasswordValidateDto passwordValidateDto, final PageRequest pageRequest, final HttpServletResponse httpResponse){
+      final String encryptedPassword = encryptService.encrypt(passwordValidateDto.getPassword());
+      User user = userRepository.findByEmail(passwordValidateDto.getEmail());
+      
+      if(!passwordValidateDto.getEmail().matches("^([_a-zA-Z0-9-]+(\\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)*(\\.[a-zA-Z]{1,6}))?$")){
```

Comment:
Could this use the isEmailValid below?

Reply:
Agree...this should have used the isEmailValid from ValidatorUtil

if(ValidatorUtil.isEmailInvalid(passwordValidateDto.getEmail())){...